yesod-form/Yesod/Form/Types.hs

```diff
@@ -80,6 +82,17 @@ instance Data.Traversable.Traversable FormResult where
       FormFailure errs -> pure (FormFailure errs)
       FormMissing -> pure FormMissing
 
+instance Alternative FormResult where
```

Comment:
Can you add a `@since` comment here, bump the minor version in the cabal file, and update the changelog?